src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -186,6 +186,17 @@ class GenericClient[F[_]](
   lazy val labels = new Labels[F](requestSession)
   lazy val relationships = new Relationships[F](requestSession)
 
+  lazy val wellDataLayerRequestSession: RequestSession[F] =
+    RequestSession(
+      applicationName,
+      uri"$uri/api/${apiVersion.getOrElse("playground")}/projects/$projectName",
+      sttpBackend,
+      authProvider,
+      clientTag,
+      cdfVersion
+    )
```

Comment:
I think we don't need this, you just have to use a different GenericClient parsing `apiVersion = Some("playground")` 😁. Since it uses a different apiVersion, I think you can create a new client and call it `playgroundClient`?

Reply:
I'm not sure I understand how the `apiVersion` argument can be used. If I set it to `v1`, then the WDL SDK will fail, since WDL is not in `v1`. However, if I set it to `playground`, then I assume that all the other resources (assets, events, etc) will start to fail, because the SDK is targeting `v1` and not `playground`.
Am I wrong?

What if I create a separate class called `PlaygroundClient` instead of `GenericClient`, and then put the `wdl` there?
I think that might be a better separation.

Reply:
I mean you can create a separate client for WDL, like what we did for [DMS by using cdfVersion alpha](https://github.com/cognitedata/cognite-sdk-scala/blob/98e3a73182fcff9c052413812f89d0f0b77a5fd6/src/test/scala/com/cognite/sdk/scala/v1/CommonDataModelTestHelper.scala#L40-L48)
So smthing like this
😁 
```scala
lazy val playgroundClient = new GenericClient[IO](
    "scala-sdk-test",
    project,
    baseUrl,
    authProvider,
    Some("playground"),
    None,
    None
  )
```

Reply:
My point is that the API in `playground` and in `v1` is not necessarily the same. And the `GenericClient` has to be made for one of them. Right now WDL only works in playground, so it fells like a weird gotcha that the GenericClient has to be initialized with `playground` to use WDL, while it must _not_ be initialized with `playground` to use assets, for example.

But I don't think it's that big of a deal either. I can change it :)

Reply:
@vuhainguyen I've made some changes here: https://github.com/cognitedata/cognite-sdk-scala/pull/587/commits/44b7453a6bfe5ff57bd7aefd6d4d0e5e106b6902

I'll be happy to jump on a call to discuss if you'd like.

Reply:
Had a talk with @vuhainguyen offline, and we agreed that it's ok to keep it as it is just now:
```scala
  lazy val wdl = new WellDataLayer[F](
    RequestSession(
      applicationName,
      uri"$uri/api/playground/projects/$projectName",
      sttpBackend,
      authProvider,
      clientTag,
      cdfVersion
    )
  )
```
since it will be easier to use, and will simplify the transition from playground to v1 for WDL.